pkg/policy/rule.go

```diff
@@ -209,6 +211,28 @@ func mergeL3(ctx *SearchContext, dir string, ipRules []api.CIDR, resMap *L3Polic
 	return found
 }
 
+func computeResultantCIDRSet(cidrs []api.CIDRRule) []*net.IPNet {
```

Comment:
Could you return `[]api.CIDR` here instead?

Reply:
Fixed.